doc/source/skips/1-governance.rst

```diff
@@ -157,7 +157,7 @@ are made according to the following rules:
   decision-making process outlined above.
 
 - **Changes to this governance model or our mission, vision, and values**
-  require a :ref:`skip` and follow the decision-making process outlined above,
+  require an :ref:`skip` and follow the decision-making process outlined above,
```

Comment:
Thanks! It should say "require a SKIP", but the text renders as "require a ... many words here ....".

Reply:
It should say "requires a SKIP" you mean ?

Reply:
No, the ["an" is used only if the following word is not started by a consonant sound](https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an). Here "SKIP" starts with a consonant sound, so "a" should be correct.

Also, I think "require" (not "requires") is the correct term as it references the plural "changes". The "s" is only used for singular third person.

I'm not a native speaker mind you, so my gut feeling and explanations may be totally off. :sweat_smile:

Reply:
So I agree with what Stéfan suggests: 
```
require a :ref:`SKIP <skip>`
```